### ISSUE TITLE
Accuracy loss problem？

### ISSUE THREAD

**Issue by xinsuinizhuan** [ultralytics/yolov5#5891]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

# Usage:
        #   PyTorch:      weights = *.pt
        #   TorchScript:            *.torchscript
        #   CoreML:                 *.mlmodel
        #   TensorFlow:             *_saved_model
        #   TensorFlow:             *.pb
        #   TensorFlow Lite:        *.tflite
        #   ONNX Runtime:           *.onnx
        #   OpenCV DNN:             *.onnx with dnn=True
        #   TensorRT:               *.engine

how about this methods accuracy loss?

**Comment by glenn-jocher** [comment]
@xinsuinizhuan val yourself:
```
python export.py --weights yolov5s.pt --include onnx ...

python val.py --weights yolov5s.onnx
...
```
